packages/tab-bar/src/mwc-tab-bar.ts

```diff
@@ -80,6 +80,24 @@ export class TabBar extends BaseElement {
     this.mdcFoundation.handleKeyDown(e);
   }
 
+  constructor() {
+    super();
+
+    const updateComplete = this.updateComplete;
```

Comment:
This reference becomes stale after the first update doesn't it? The `this.updateComplete` getter will return different promises at different times so that it always points to a promise that resolves for the _next_ update